jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -874,17 +881,21 @@ private void openSession() throws SQLException {
     }
 
     // set the session configuration
-    if (sessConfMap.containsKey(HiveAuthConstants.HS2_PROXY_USER)) {
-      openConf.put(HiveAuthConstants.HS2_PROXY_USER,
-          sessConfMap.get(HiveAuthConstants.HS2_PROXY_USER));
+    final String hs2ProxyUser = sessConfMap.get(HiveAuthConstants.HS2_PROXY_USER);
+    if (hs2ProxyUser != null) {
+      LOG.debug("Set hive.server2.proxy.user: {}", hs2ProxyUser);
+      openConf.put(HiveAuthConstants.HS2_PROXY_USER, hs2ProxyUser);
     }
 
     // set create external purge table by default
     if (sessConfMap.containsKey(JdbcConnectionParams.CREATE_TABLE_AS_EXTERNAL)) {
       openConf.put("set:hiveconf:hive.create.as.external.legacy",
           sessConfMap.get(JdbcConnectionParams.CREATE_TABLE_AS_EXTERNAL).toLowerCase());
     }
-    if (isHplSqlMode()) {
+
```

Comment:
I'd suggest to log out in debug level the whole oponeconf right before opening the session, instead of having a log.debug next to each set command, in a similar fashion as you log out the serverHiveConf after it is opened.

Reply:
Ya.  I wanted to log them differently since they have different sources (and I wanted to preserve the source), but I guess it uses `set:hiveconf` `set:hivevar` in the key property so the source will be known.  Thanks. Changed.

I did keep the 'table name' since that is just all around useful information that I wouldn't want to go digging for in the big configuration map.